### DISCUSSION TITLE
Esc to deselect tool instead of exiting program

### DISCUSSION THREAD

**Discussion by dorkcooldork** [flameshot-org/flameshot#2722] [Q&A] [answered]
Hi there is there a possibility that after typing in a textbox, you press Esc and the content of the textbox still stays but the text tool is deselected? So basically, when you press Esc, instead of deleting the whole textbox you are currently typing on (and still haven't clicked out of), it just changes to the arrow tool?

**Comment by mmahmoudian** [comment]
If this is a feature request, I would be thankful if you post it as such in the issues section. Thank you.

**Comment by mmahmoudian** [comment, marked as answer]
I don't remember when this is implemented, but in Flameshot v12.1.0 (57c8e8d8) it works as you asked for. When Esc is pressed, it first checks if a tool is active and will deactivate that, if not, it will terminate screenshot
